### PR TITLE
Tweak default ship and variant loadouts to reflect missile changes by @Pointedstick

### PR DESCRIPTION
**Pull request by MCOfficer**
Ref: endless-sky/endless-sky#3922

> 
> # Changes
> 
> This PR changes the default loadout of many ships and variants to reflect the logical conclusions of the recent missile tweak and improve in-game realism, fun, and tactical opportunities.
> 
> In a nutshell, here are the changes:
> 
>   * Most ships that previously had no missile armament now have some according to their typical default battle roles:
>       
>     * Interceptor/fighter: meteor, sidewinder, or javelin launchers
>     * Interceptor-killer/anti-fighter defense: sidewinder missile launchers
>     * Capital ship bombardment: torpedo launchers
>     * Multirole/pirate/anti-merchant: meteor missile launchers
> 
>   * Nearly all fighters, interceptors, and light warships have a Small Radar Jammer. [In the event that these outfits ever become useful](https://github.com/endless-sky/endless-sky/issues/3912), they should allow these ships to have a modicum of a chance against sidewinder missiles. It seemed logical that countermeasures against anti-fighter missiles would be more or less standard equipment.
> 
>   * All turreted ships that previously lacked an anti-missile turret got one. Even before my missile changes, it was always odd how some large slow ships like Modified Argosies let you pound them to death with Meteors at long range with no defense whatsoever. No more. Large capital ships generally mount two Heavy Anti-Missile Turrets--one of the ways you can distinguish them from smaller lighter warships.
> 
>   * Added a number of new variants that are heavily armed with missiles. Check out the `Vanguard (Missile)` variant, which is basically an [arsenal ship](https://en.wikipedia.org/wiki/Arsenal_ship).
> 
>   * More fleets include heavy missile-armed interceptor and light warship escorts.
> 
>   * Gave the Hawk 10 more weapon space and reduced its Cargo space by 10. This was the only way I was able to make the Hawk actually excel at anything over a Berserker or a Fury. With 50 tons, they can carry a mix-and-match assortment of Plasma cannons and Torpedo Launchers, making them truly useful bomber craft.
> 
> 
> # Testing
> 
>   * Played through the entire Free Worlds campaign in various ships, some armed with missile weapons, and some armed exclusively with guns and turrets.
>       
>     * Early game was no more difficult than it was before with all three ships
>     * No major change to the overall balance during the campaign
>     * In some cases, short-range battles are a bit slower because of the replacement of guns and gun turrets with missile weaponry. I think this is a good thing overall, since fleet battles in Endless Sky are generally over very quickly.
> 
>   * Made sure all changed ships and variants could fit their new loadouts, and that their energy and heat profiles still made sense. Made minor tweaks to preserve these where appropriate.
> 
> 
> # Feelings
> 
> These changes make the game feel more tactically interesting to me. Now that missiles are a genuine option and most ships are generally armed and defended accordingly, player armament choices are no longer a no-brainer: will you forget about missiles and close to knife fighting distance, taking advantage of your ship's superior short-range gun armament? Will you mount missiles to all of your gun ports and bombard your enemies from a distance, oversaturating their anti-missile defenses? Will you play it safe and be balanced, with some guns and some missiles? Fighters are actually more fun to play with now, since they're basically flying missile platforms, which can be very effective.
> 
> Overall I believe this PR improves the game's fun factor and realism, and opens new tactical possibilities.

### CODE REVIEW

**Comment by Amazinite** [comment]
Labeled considering because it's a balance change.

**Comment by KiLEdEnNis** [comment]
at some point before the update we should retest the story's combat missions (human and Wanderer) anyway, so these changes may make it more fun

**Comment by Pointedstick** [comment]
As I mentioned, I went through the entire human campaign with these changes to test them. :)

**Comment by KiLEdEnNis** [comment]
yes, these changes, not all changes (future too) compared to the last Steam build that'd affect the battles, for both this and the Wanderer campaign

**Comment by Pointedstick** [comment]
Well, I did the entire human campaign with this patch plus everything else from git master, which includes everything since the last steam build. The Wanderer campaign is mostly unaffected since you're not generally fighting Human ships, and alien ships' default armament was not touched in this PR.

**Comment by KiLEdEnNis** [comment]
I'm saying that there may be more changes before the next update, not just to missiles, that affect all battles in the stories (i.e. some Wanderer story battles already have to be rebalanced, think I made an issue for it)
so I don't think it being a balance change necessarily means it has to wait to be merged (Ama giving it a considering tag), since before the next release we'll make sure it's do-able enough
but it doesn't really matter much

File: data/jobs.txt [hunk 1]
```diff
@@ -1069,9 +1069,9 @@ mission "Escort (Tiny, Southern, Dangerous Origin or Destination)"
 			variant
 				"Sparrow"
 			variant
-				"Wasp"
+				"Fury"
 			variant
-				"Berserker"
+				"hawk"
```
Comment (KiLEdEnNis) [inline data/jobs.txt]: *Hawk

**Pull request closed without merging**